changelog_generator/__main__.py

```diff
@@ -0,0 +1,107 @@
+import os
+from typing import List, NamedTuple
+
+from jinja2 import Environment, FileSystemLoader
+
+from changelog_generator.commit import Commit
+from changelog_generator.repository_manager import RepositoryManager
+
+
+class CommitTree(NamedTuple):
+    commit_type: str
+    commits: List[Commit]
+
+
+def render_changelog(
+    organization: str,
+    repository: str,
+    previous_tag: str,
+    current_tag: str,
+    commit_trees: List[CommitTree],
+) -> str:
+    template_loader = FileSystemLoader(searchpath=os.path.dirname(__file__))
+    template_environment = Environment(loader=template_loader)
+    template = template_environment.get_template("changelog_template.jinja")
+
+    return template.render(
+        organization=organization,
+        repository=repository,
+        previous_tag=previous_tag,
+        current_tag=current_tag,
+        commit_trees=commit_trees,
+    )
+
+
+def get_commit_from_type(commits: List["Commit"], commit_type: str) -> List["Commit"]:
```

Comment:
Why the " around `Commit` ? Same for the rest of the type hints throughout this module